apps/org/utils.py

```diff
@@ -17,8 +20,25 @@ def set_unique_slug(instance, based_on_field='name'):
     slug = slugify(getattr(instance, based_on_field))
     # If this slug is already being used, create a more unique slug
     if instance.__class__.objects.filter(slug=slug).exists():
-        slug = '{}-{}'.format(
-            slug,
-            ''.join(random.choices(string.ascii_letters + string.digits, k=20))
-        )
+        slug = '{}-{}'.format(slug, ''.join(
+            random.choices(string.ascii_letters + string.digits, k=20)))
     instance.slug = slug
+
+
+def make_qr_code(text, format='PNG', encoding='base64'):
+    """
+    Convert the given text to a QR-code. If encoding=='base64', return base64;
+    otherwise, return raw image file data as bytes.
+    """
+    qr = qrcode.QRCode(version=None)
+    qr.add_data(text)
+    qr.make(fit=True)  # is a Pillow Image
+    image = qr.make_image(fill_color="black", back_color="white")
+    f = io.BytesIO()
+    image.save(f, format=format)
+    f.seek(0)
+    b = f.read()
+    if encoding == 'base64':
```

Comment:
Given the behavior of returning base64 or raw, the kwarg `base64=true` is more clear, preventing devs from trying other encodings that don't exist and will have the raw data returned unexpectedly.

Reply:
Good idea